src/components/survey/Survey.tsx

```diff
@@ -0,0 +1,90 @@
+"use client"
+import { ChangeEvent, useState, useCallback } from 'react';
```

Comment:
I believe that both `ChangeEvent` and `useCallback` are unused, but I reckon this is something that you wanted try and implement at some later point?